README.md

```diff
@@ -1175,6 +1349,40 @@ NOTE: `{% csrf_token %}` tags need access to the top-level context, and they wil
 
 Components can also access the outer context in their context methods by accessing the property `outer_context`.
 
+## Rendering JS and CSS dependencies
```

Comment:
Moved the section on middleware here, as I expect that this section will be expanded when we tackle https://github.com/EmilStenstrom/django-components/issues/478